fastestimator/backend/sparse_categorical_crossentropy.py

```diff
@@ -12,20 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from typing import TypeVar
+from typing import Dict, Optional, TypeVar
 
 import tensorflow as tf
 import torch
 
 from fastestimator.backend.reduce_mean import reduce_mean
 
 Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
+Weight_Dict = TypeVar('Weight_Dict', tf.lookup.StaticHashTable, Dict[int, float])
 
 
 def sparse_categorical_crossentropy(y_pred: Tensor,
                                     y_true: Tensor,
                                     from_logits: bool = False,
-                                    average_loss: bool = True) -> Tensor:
+                                    average_loss: bool = True,
+                                    class_weights: Optional[Weight_Dict] = None) -> Tensor:
```

Comment:
we should have end user just pass a regular dictionary regardless. we can convert it to the staticHashTable behind the scenes. Torch might also have an equivalent that we can use to get batched lookups

Reply:
As it's called in forward, won't we be creating staticHashTable every batch instead of once?